.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Declare that all files will always have LF line endings on checkout.
+* text eol=lf
```

Comment:
if the readme section still required then?

Reply:
Good point